sw/device/silicon_creator/rom/rom.c

```diff
@@ -217,7 +217,6 @@ static rom_error_t rom_init(void) {
 
   // Store the reset reason in retention RAM and clear the register.
   retention_sram_get()->creator.reset_reasons = reset_reasons;
-  rstmgr_reason_clear(reset_reasons);
```

Comment:
Curious: Do we not want to clear that the POR happened, so we don't trigger any POR-specific code in ROM or ROM_EXT on deep sleep wakeups or NDM resets?

Reply:
This is a request from the prodc customer to not clear the reset reason in ROM - they want to check the register and clear it in their firmware.  This draft PR is an experiment to see if there is any test fallout.

We'll then control this behavior via an OTP so the sku_creator can decide how they want the ROM to behave.

Reply:
This sounds a little sketchy to me. However, I'll not stick my nose further into this, hehe.

Instead, I'd just tag @mundaym for visibility, since it seems like there might've been at least some discussion at one point: https://github.com/lowRISC/opentitan/issues/7887

Reply:
Ah, was asked to tag @felixmiller too